### DISCUSSION TITLE
ASA Stats NFT Collection

### DISCUSSION THREAD

**Discussion by wxivd** [asastats/channel#111] [Ideas]
This discussion started in [ASA Stats Deep Liquidity issue #62](https://github.com/asastats/channel/discussions/62#discussion-4058852), and moved here based on @ipaleka [advice](https://github.com/asastats/channel/discussions/62#discussioncomment-2836115).

### ASA Stats NFT Collection
Creating our first NFT collection. This is a dual solution aims to increase liquidity by ASA Stats project fund and keen investors.
1. **NFT advantages for ASA Stats project**
* Raise funds from NFT sales to provide deep liquidity to our LP pools (current/future). Expediting project development.
* Giveaway for active community members incentivising community contribution.
* Marketing, Increase awareness of ASA Stats.
2. **NFT advantages/Utility for investors/community members**
* Staking Rewards for NFT holders: APR to holders paid in ASA Stats Tokens. Giving rewards to NFT holders is similar to liquidity provider incentive but we get to decide fund allocation in LP pools. NFT staking rewards runs for the duration of LP program.
* Free NFT drop to LP providers that satisfy additional criteria: X amount of LP tokens above ceiling limit + LP locked for specific duration
* Free advanced ASA Stats features to choose from once these are rolled out (1 free advanced feature per wallet, non-commercial use).
3. **NFT Creation process**
* Assign Budget NFT creation Cost, the blockchain require to have 0,1A per NFT blocked on the creator's account + 0,001 for creating the asset. Artist commission: depends on reputation, ~month salary where he's come from, and
* Number of NFTs
* NFT Theme (art technique, traits)

**Initial thoughts**
* Estimated total cost: 1500-3000 Algo. We can negotiate a deal that include % of the sales for artist, a given number of NFTs (thus optimising cost).
* Total number 350 NFTs
* Minting price 150-200A
* NFT Theme, Generative art, Stats for Statistics: have the following traits: Category: Descriptive Statistics Types: measures of frequency, central tendency, dispersion or variation, position. Visual art (graphs, charts, pies, histograms, scatter plots..etc) Description: basic statistics (educational value)

**Comment by wxivd** [comment]
I c/p previous [comments](https://github.com/asastats/channel/discussions/62#discussioncomment-2836115), for context

by @Motuwagon in [Discord](https://discord.com/channels/906917846754418770/973245225680109588/979733486686572544);

> We were talking about a Asa Stats nft a while back. I am not a artist but anyone like the idea of Alpaca nft as a play on the team leads name.

by Mayor | CM YouTube
https://discord.com/channels/906917846754418770/973245225680109588/979767920240496690

> I'm interested in working with @ipaleka on an ASA series

By @Motuwagon
> Quick google search there is a eth nft Alpaca project and a financial crypto company that uses name and mascot Alpaca and advertising their api for stock and crypto. so maybe it would be a poor choice. But did not find a Algo one yet. Image search they are cute at any age. With unique hair cut can make for fun personality.

![170777604-17dbd766-efa2-4cd8-801d-fb9d8bec1f39](https://user-images.githubusercontent.com/95645978/170793012-84b21605-e8a1-4a7b-a00e-9814166870c2.png)

**Comment by ipaleka** [comment]
by motuwagon in [Discord](https://discord.com/channels/906917846754418770/914540797054025788/949459181142937601):
- - - - -

I like Psyspira work.  Original abstract. 
 https://www.randgallery.com/collection/tapestries/

https://www.nftexplorer.app/collection?creator=XY5HTN2EYICTR6VZFX2D6MCLZYR4OMQWBOKPIEZC5BZSYJ6DY6J5BUQ4YM

- - - - -

**Comment by AngelsOfAres** [comment]
Dear ASA Stats,

We at Angels Of Ares would like to extend an offer to help provide the art aspect for your upcoming plans. We have 3 artists varying in their skill set, and therefore can produce art based on your needs and criteria.

Feel free to reach to us on Discord, Twitter, or Telegram and we would be happy to have a further discussion.

You may find us via our website https://angelsofares.xyz

**Reply by ipaleka** [reply]
Hi! I'm so glad you reached us here. For the others, AngelOfAres has been a member of our community and as we collect NFT data from the blockchain, the name has popped up in our logs every now and then... :)

We'll now wait for some suggestions from our community, I guess we all want to see what @SCN9A and @motuwagon wish to say about this. :)

**Reply by Motuwagon** [reply]
NoIm poking around nftexplorer and there discord a bit.  
I personally like the mind bender series they have.  A.i. style ART vibrant colors.
If There is consensus what it is wanted generally then suggest ask for one off originals from the art community in the socials we already use.  Head hunt and invite specific artists to participate and open call.  Based on our criteria.  Have a vote on style and form best that accomplished the vision.  This should stimulate community and awareness of up coming nft series.  Reward top 10 artist that participate with Stats based on total votes and winner with stats and contract for the final 349 pieces.

Criteria possibilities
1. Original work with no copyright infringements! Willingness to fulfill the winning contract 
2. Originality Easily identifiable, on a thumbnail view we can pick out Asa Stats nft at a glance
3.  Easily attributable to the Stats project, make some sense why/how it is Stats nft. 
4. Existing Algo generative nft minting experience, adding arc data

Any artist or group willing to take on the commission work that produces quality pieces I'm for.

**Reply by AlgoRhythMatic** [reply]
> Have a vote on style and form best that accomplished the vision. This should stimulate community and awareness of up coming nft series. Reward top 10 artist that participate with Stats based on total votes and winner with stats and contract for the final 349 pieces.

Sounds like a great plan!  As noted in a comment above - I really like your idea re: Alpaca as a light-hearted mascot.  I also like the idea of some statistics related entries per @SCN9A - assuming artists can make them engaging and/or entertaining.  Any other categories you would consider?

**Reply by wxivd** [reply]
This is exciting and I am loving the engagement we are having for our main NFT collection. It is a subject of interest for all of us and definitely reflects our meaningful connection to ASA Stat. I welcome all ideas. There is much room for innovation and artistic ideas.

I definitely agree with @Motuwagon 
Rather than choosing one NFT creator right away, I recommend that we agree on the particular aspects that help NFT artists assessing this opportunity, the creating of ASA Stats NFT. From my initial research on this. some artists do have the skill but do not have the time for large collection (i.e. number of NFTs, art technique). 

These aspects relate to the time and expertise that are required, the incentives as well. Once we define these criteria, we can head-hunt or create an-open invitation to NFT artists on social media / forward it to our favourite creators. Providing that this opportunity is sponsored by ASA Stats, we get a selection of keen artists to choose from. 

For the theme, I really like the idea and I am all for it.
> A alpaca sir for ipaleka.

**Reply by Motuwagon** [reply]
> I definitely agree with @Motuwagon Rather than choosing one NFT creator right away, I recommend that we agree on the particular aspects that help NFT artists assessing this opportunity, the creating of ASA Stats NFT. From my initial research on this. some artists do have the skill but do not have the time for large collection (i.e. number of NFTs, art technique).
> 
What might work out, if a smaller series is more achievable.  Breaking up the series into 50, 100, 150, and 50 for instance. Stats could choose multiple artists.  If multiple really good artists submit nice work, having those names linked to the nft would be great. Also giving a chance to put levels or catagories in the nft collection or stretching release over time.  If that was a direction needed or wanted giving flexibility. 
Artist could enter the size collection they are willing to create.
  Or have multiple artist search periods. One to choose first 50 then second contest for 100 etc. Which ever number servers it's purpose.

**Reply by Motuwagon** [reply]
>  Any other categories you would consider?

Categories
-Cute mascot pfp with charts in background
"ASA Animals" ?
-Informational
-Abstract 
A piece that shows how complex watch gears are and a face that is simple  looks like a chart hour minute second hand.  To draw a line on how simple Asa Stats is to use but how much work is under the surface.
  That kinda thing.

**Reply by Motuwagon** [reply]
<img width="750" alt="VVHID6ZMKVH5JGX3MXFMEOFB7I" src="https://user-images.githubusercontent.com/103940952/174695415-33a31caf-8411-4927-b82f-974961621545.png">


I like this layout for a nft , nft can have a layer of pfp (Silvio /Alpaca) pointing at chalk board.  Then the chalk board can be anything, informational, funny , abstract, Algo symbol, Asa Stats logo etc.
Artists can do smaller limited runs if needed while keeping overall look.

**Comment by AlgoRhythMatic** [comment]
I'm a bit late on this one, but love the idea - NFTs are still a segment I am quite light on.  On Algorand chain, I only have "The Governor" so far.

> NFT Theme, Generative art, Stats for Statistics: have the following traits: Category: Descriptive Statistics Types: measures of frequency, central tendency, dispersion or variation, position. Visual art (graphs, charts, pies, histograms, scatter plots..etc) Description: basic statistics (educational value)

I like the idea of some imagery related to "stats" - but think it would also be nice to have some more light-hearted entries too.  I particularly like @Motuwagon's suggestion of Alpaca as a mascot of sorts (as a play on Ipaleka).

**Reply by Motuwagon** [reply]
If a talented artist took all the confused math gif replaced main character with a alpaca sprinkle in a pie chart made from a cheese wheel I would be sold lol.  A alpaca sir for ipaleka.

**Reply by ipaleka** [reply]
I'm full for the idea of some motor wagon NFT like moturghini. :)

**Comment by ipaleka** [comment]
Just a side note that no matter how many collections and NFTs we create, the most valuable three NFTs are reserved for TresTres, Investordooh, and @dragmz as defined back in December 2021 (DragMZ replaced my position).

**Reply by Motuwagon** [reply]
Some aspects of value will be out of Stats control.  Rarity can be controlled and size of collection also the ownership benefits can be adjusted to help give one nft value over another.  Once they are on the market value becomes subjective.  The market may decide artistic value out weighs benefit value or rarity.  If tres, dooh, and drag, picked which they liked the most made those  nft 001, 002, 003 in series, adjusted arc data to raise rarity most likely it would equal most valuable.  At least if they picked it would be the ones they found to be the most valuable at time of minting.

Those 3 could also have a weighted vote be the ones who decide which artist Stats use.  If it does go in direction of voting I'd suggest restricting voting to some capacity.  I see ballot stuffing in other groups getting random people to vote based on who entered not what, in most competitions and a popularity contest takes away from the art.  Perhaps making voting open to Lp providers and Governors.  Keep it in house.  I see it as stats asking freelance artists to do contract work and Stats judging their portfolio, experience and provided submission. To award the contract.

**Reply by ipaleka** [reply]
Thanks for another set of valuable insights and suggestions! :)

There will be no chance involved in preparing and picking NFTs for those three. If we can't do that differently, then we'll create just those three NFTs and send them to the owners. 

This suggestion has been brought by @SCN9A as a try to solve our liquidity issues and I see - in front of the Team - that issue to be less significant than the fact that the Team hasn't yet provided promised NFTs to those two (and DragMZ afterward).

**Comment by Motuwagon** [comment]
I was thinking of a feature that might help it's nft sale.   And hopefully if this works show how Stats is a cut above. A while back there was discussed about a minimum floor being set on a nft.  
Assuming the value of Asa Stats will increase over time. 
 Since the purpose is to add liquidity. And I don't want to mess that up.  Would it work to use half of proceed of each sale to buy Asa stats and lock them in a smart contract to the  nft sold to guarantee a buy back of nft at half it's current value in Asa stats  at time of purchase.  Making nft like a savings bond.   After the LP payout ends it will hold and might gain value as Asa Stats grows in value.  While stats already bought the eventual payout at the beginning when prices are lower.
This  feature could pass from one owner to the next if resold on open market but only at the locked in amount not at secondary sale price.  Would be a clever feature I don't see with nft's.  Then burn feature to unlock stats would send nft to mint wallet and trigger the smart contract to send stats to wallet that burned nft.  Perhaps connecting  payout to time held.  10% original cost if burned within a year up to 50% if burned after dao accomplished.

Side note a bonus to entice artist secondary sale royalty percentage.  Nft should be minted on a stats wallet royalties split between stats and Artist

**Reply by ipaleka** [reply]
I can't be involved right now in a deeper discussion about your - for sure valuable - suggestions, I'll just comment on a few things.

> Would it work to use half of proceed of each sale to buy Asa stats and lock them in a smart contract to the nft sold to guarantee a buy back of nft at half it's current value in Asa stats at time of purchase.

ASA Stats doesn't own the devpool funds, they are completely under control of the Team and the Team can only borrow those funds to ASA Stats and other pools.

The other way how ASA Stats can get fiat currency and ALGO funds is to sell its own ASASTATS. 

That being said, ASA Stats can donate ASASTATS for this suggestion, while buying them on the market isn't an option.

> Side note a bonus to entice artist secondary sale royalty percentage. Nft should be minted on a stats wallet royalties split between stats and Artist

The following is only my personal stance on this: I'd rather that we stay aside and leave those royalties to the real owners as IMO that is better for our branding.

**Reply by Motuwagon** [reply]
Secondary sale royalties are set at the site facilitating the sale.  Rand has it set at 7.5%.  while algoxnft let's seller adjust the royalty. This percentage will go to wallet that created the nft either artist or Asa Stats wallet.  For simplicity of the other moving parts I suggested.  Setting it to a ASA Stats wallet would be more efficient.  I agree with giving the artist the royalties make their effort more enticing.  If they can reep % everytime any of the nfts sell after the initial release.

**Comment by ipaleka** [comment]
https://github.com/asastats/channel/discussions/178#discussioncomment-3722422

- - - - -

**ASA Stats Utility NFT**

As we know, words about NFTs spread like crazy. If it has a utility, even better. We can initiate a creation of a utility NFT. 
I was picturing something like this. They can be bought with ASASTATS token(after whitelist spots etc.)

Wallets holding a...

**Common**(30 items) for a price of 50 Algo equivalent:

- can change the appearance of the app(skins, etc.)


**Uncommon**(10 items) for a price of 150 Algo equivalent:

- common features + get real-time total value updates(no refresh needed)


**Legendary**(2 items) for a price of 1500 Algo equivalent:

- get a life time subscription and a governor seat


If you like or support the idea, please add your suggestions, or give an opinion if you don't. We can hire a NFT creator or there maybe is one in our community?  We can also make a twitter giveaway for an NFT. ****

**Reply by ipaleka** [reply]
By @marucat27

I do believe the creation of NFTs will especially with utility will gain attention of the community. Mint size will be important also. Low mint count will for sure make it desirable but with less total funds received. What subscription tier were you thinking of the legendary NFT?
- - - - -
By @kerrilija

Everything is to be discussed. We need to find a balance between being generous and not doing damage to our services. We have to calculate what is the cost of having a subscripted user (how much resources is he or she leveraging), so we can be more accurate in terms what premium features can we offer for free. I've put random values/numbers. Maybe you or someone with more experience in NFT in general can provide better item numbers, rarity options etc.

**Comment by kerrilija** [comment]
By @ipaleka 

> Last but not least, this doesn't mean that your idea is bad, at contrary, but we have to bypass [The](https://github.com/asastats/channel/wiki/ASA-Stats-Token-utilization#the-first-dao-voting-session) condition somehow, for example by staking such an NFT in order to get ASASTATS used to purchase a subscription and that way accomplish what you have proposed._


A holder of a certain NFT, gets "accelerating staking" bonus(+70% rewards or something like that). I'll just put some random numbers. If someone buys an NFT (paid in ASASTATS), and there is an amount of no less than 0.4 damo in a period of one full month, he/she gets rewarded with 0.3 damo. If the amount in the wallet goes below 0.4 damo or the NFT leaves the wallet in that period, he/she gets no rewards. So to quailfy for the "accelerated staking" user has to be a holder of an ASA Stats NFT. Various rarity can be for various reward percentages...

**Reply by ipaleka** [reply]
> If someone buys an NFT (paid in ASASTATS)

Just a remark that this shouldn't be a condition as there's no practical reason behind it like in the case of Permission ASC. Restricting to ASASTATS would be counterproductive and we surely have to facilitate the well know existing NFT markets to sell our NFTs for all the currencies those NFT markets allow.

> damo

Another remark, damo is the **community rewards** unit. If someone [contributes to the project](https://github.com/asastats/channel/discussions/250) they receive damo. No other usage for that term/unit exists.

> If someone buys an NFT (paid in ASASTATS), and there is an amount of no less than 0.4 damo in a period of one full month, he/she gets rewarded with 0.3 damo. If the amount in the wallet goes below 0.4 damo or the NFT leaves the wallet in that period, he/she gets no rewards. So to quailfy for the "accelerated staking" user has to be a holder of an ASA Stats NFT. Various rarity can be for various reward percentages...

Your logic is clear and straightforward, to reward our NFT holders, tho it implies, as it seems, we'd use our facilities for staking them, and that probably won't be the case. Our resources are restricted and our own incentive program has been an exception to the rule, in the future we'll use other providers in the ecosystem for the purpose.

So, the plan is good, by combining staking of our NFT and ASASTATS staking, based on the amount of ASASTATS and NFT rarity, such a user can get various levels of rewards, respectively different subscription tiers. I guess we should also allow just staking of our NFT (without any additional ASASTATS) and put that level of staking somewhere on the staking scale to be comparable to staking just ASASTATS.

**Reply by kerrilija** [reply]
Oh, 1 damo was equal to 1,000,000 ASASTATS in my mind. Sorry for that :D
I know it requires additional work to handle the staking by ourselves. But how will the staking platform(Cometa for example) give more rewards our NFT holders? That would be another contract, another dedicated staking pool, with a prerequisite - the NFT. If it's possible, great.

**Reply by ipaleka** [reply]
> Oh, 1 damo was equal to 1,000,000 ASASTATS in my mind.

You don't need to apologize, I just wanted to emphasize that damo is used as a measure of community contribution. :)

> I know it requires additional work to handle the staking by ourselves. But how will the staking platform(Cometa for example) give more rewards our NFT holders? That would be another contract, another dedicated staking pool, with a prerequisite - the NFT. If it's possible, great.

Yes, the simplest approach, I guess, would be to stake the NFT itself. Tho we should check for other variants, I just don't think that producing our in-house solution would bring any benefits.

**Reply by marucat27** [reply]
I believe there are NFT staking solutions currently.  Flippingalgos.xyz (Scott Marin) also NFT creator of Billy Big Beaks collection offers NFT staking solutions.  I don't know if there are others.  But I suppose you could probably identify if the user has the NFT in their wallet at certain snapshots of the day or week to be eligible for rewards/subscription services.  The idea of NFTs will get a wider audience interested I feel but definitely we want them to hold the NFT and re-stake said rewards back or contribute back.  I like the idea of staking because it makes the NFT a little more sticky and just a little more inconvenient for a user to just list on secondary like Rand.  I know some users will hold NFTs for air drop snap shots say until Friday night and after receiving the air drop will list the NFT on Rand.  This happens with like the zone afk elephants and alchemont NFTs.

**Reply by marucat27** [reply]
A random thought regarding NFTs.....what if the NFT could be almost like a profile pic of your ASA Stats Page?  There could be a a place in your ASA Stats page to display the NFT almost like a badge to show your support for the project.  I think there's a possibility that something like that could gain interest but then again it might disrupt the cleanliness of the ASA STATs page which I feel is its strength too.  Or maybe not a profile picture but It could show you level of support like say Level 1, or Level 2, or Level 3 at highest. I think Vestige has something similar to this with regards to holding amounts of vest tokens like X amount for fish, then for a shark, whale, etc. People often will strive to hit certain token amounts or tiers.  I personally was content with Defly's X tier but when I saw an Alpha Tier opened up it gave me a goal to strive for.

**Reply by ipaleka** [reply]
> I know some users will hold NFTs for air drop snap shots say until Friday night and after receiving the air drop will list the NFT on Rand. This happens with like the zone afk elephants and alchemont NFTs.

Imo we shouldn't be afraid or prepare a mechanism against this. If they sell our NFT that means there's demand for it and in this particular case that could mean that the buyer seeks a discount to purchase our subscription tier! :)

**Reply by ipaleka** [reply]
> A random thought regarding NFTs.....what if the NFT could be almost like a profile pic of your ASA Stats Page?

Another example of a user widget we can sponsor the development of... As in the previous example in our Discord, feel free to add it to the related [wiki page](https://github.com/asastats/channel/wiki/FeaturesDependentOnUserWidgets). :)

Once we get user widgets my opinion is that we should always default to an unselected widget in the user settings. If they want that *badge* to be shown (that we're going to sponsor development of and afterwards it will be available to select on the user settings page) then they should fist confirm that by selecting that widget.

**Reply by ipaleka** [reply]
> im happy to provide staking solution(s) for this project/idea or any others or even develop something custom just for asastats.. im around

That's really good to know, thank you!

Have you read our utilization document and about the plans for the development of our [Permission ASC](https://github.com/asastats/channel/wiki/ASA-Stats-Token-utilization)?
